README.org

```diff
@@ -34,6 +34,30 @@ It is recommended you read Nixpkgs and NixOS documentation on package overlays a
  - https://nixos.wiki/wiki/Overlays
  - https://nixos.org/nixpkgs/manual/#chap-overlays
 
+*** Adding the overlay to an existing Flake
+This overlay can be added to a flake by adding following to your existing files:
+
+=flake.nix=
+#+begin_src nix
+inputs = {
+  emacs-community = {
+    url = "github:nix-community/emacs-overlay";
+    inputs.nixpkgs.follows = "nixpkgs";
+  };
+};
+#+end_src
+
+=configuration.nix=
+#+begin_src nix
+nixpkgs = {
+  overlays = [
+    inputs.emacs-community.overlay
```

Comment:
`inputs` is not a module input by default.

To avoid confusing for beginners, I suggest you put everything in a flake.nix file.